azure-pipelines.yml

```diff
@@ -0,0 +1,105 @@
+# ASP.NET Core (.NET Framework)
+# Build and test ASP.NET Core projects targeting the full .NET Framework.
+# Add steps that publish symbols, save build artifacts, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/dotnet-core
+
+trigger:
+- master
+
+pool:
+  vmImage: 'macos-latest'
+
+variables:
+  buildConfiguration: 'Release'
+  mdocPath: 'bin/$(buildConfiguration)'
+  
+steps:
+- task: NuGetToolInstaller@1
+  displayName: Install NuGet Tool
+
+- task: Bash@3
+  displayName: Run Unit and Integration Tests
+  inputs:
+    targetType: 'inline'
+    script: 'make prepare all check CONFIGURATION=Debug'
```

Comment:
We can just leave the `CONFIGURATION` parameter as the default since it defaults to `Release` ... however you could also pass in `$(buildconfiguration)` then that way it's configurable via the build.